repeat_expansion_variants/clinvar_identifier_parsing.py

```diff
@@ -0,0 +1,108 @@
+"""
+The regular expressions in this module are used to parse ClinVar's identifiers for describing repeat expansion variants.
+The pyhgvs module cannot be applied here because not all expression used by ClinVar are actually valid HGVS, and that
+module imposes strict validation. Hence, custom regular expressions are necessary.
+"""
+
+import re
+
+
+# Common part for all HGVS-like transcript definitions, e.g. 'NM_001256054.2(C9orf72):'
+hgvs_like_transcript_part = (
+    r'(?P<transcript_id>[A-Za-z0-9_]+)'  # Transcript accession                      NM_001256054
+    r'\.'                                # Delimiter, transcript accession/version   .
+    r'[0-9]+'                            # Transcript version                        2
+    r'[A-Za-z0-9_.()]*'                  # Gene symbol in parentheses, optional      (C9orf72)
+    r':'                                 # Delimiter, transcript/variant info        :
+)
```

Comment:
While I understand that the entire HGVS grammar might be too strict, I guess at least accessions should be consistent. So, could you ensure that this grammar is consistent with the grammar [here](https://github.com/biocommons/hgvs/blob/master/hgvs/_data/hgvs.pymeta#L207)?

Reply:
I examined HGVS grammar, and the only important difference was that the transcript accession can't start with a digit, and my previous regexp allowed that. Fixed in https://github.com/EBIvariation/vep-mapping-pipeline/pull/3/commits/83204006007f425ccfd2bf955c1cc50b4c64e285